tpm2/encoding_test.go

```diff
@@ -371,3 +371,38 @@ func TestECCParamsEncodeDecode(t *testing.T) {
 		t.Fatalf("got: %+v\nwant: %+v", got, params)
 	}
 }
+
+func TestSignEncode(t *testing.T) {
+	t.Run("AlgNone", func(t *testing.T) {
+		testCmdBytes, err := hex.DecodeString("800000010000000d40000009000001000401020304000301020300108024400000070000")
```

Comment:
How were these strings generated? Could you add a comment in case we need to update it in the future?